docs/specs/target.md

```diff
@@ -0,0 +1,182 @@
+# Target Specification
+
+This document specifies requirements for installation targets for the
+integration of Vector.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”,
+“SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be
+interpreted as described in [RFC 2119].
+
+Other words, such as "agent", "aggregator", "node", and "service" are to be
+interpreted as described in the [terminology document][terminology_document].
+
+- [1. Introduction](#1-introduction)
+- [2. Installation Targets](#2-installation-targets)
+- [3. Deployment Architectures](#3-deployment-architectures)
+  - [4. Agent Architecture](#4-agent-architecture)
+  - [5. Aggregator Architecture](#5-aggregator-architecture)
+  - [6. Unified Architecture](#6-unified-architecture)
+- [7. Hardening](#7-hardening)
+
+## 1. Introduction
+
+In its simplest form, installing Vector consists of downloading the binary and
+making it executable, but leaves much to be desired for users looking to
+integrate Vector in real-world production environments. To adhere with Vector's
+["reduce decisions" design principle], Vector must also be opinionated about how
```

Comment:
> ["reduce decisions" design principle]

Was this intended to be a link somewhere?

Reply:
Oops that got dropped between commits somehow - was intended to link to https://github.com/vectordotdev/vector/blob/master/docs/USER_EXPERIENCE_DESIGN.md#be-opinionated--reduce-decisions

Reply:
Fixed in f3f68eec90bbb4568db9023c733ee8c47e368ec7